lib/ansible/module_utils/docker/common.py

```diff
@@ -279,6 +313,85 @@ def get_connect_params(auth, fail_function):
                 timeout=auth['timeout'])
 
 
+class DockerFileStore(object):
```

Comment:
Is there a reason why you put this into `module_utils/docker/common.py`? Is this needed by more than one module?

Reply:
So far it isn't used anywhere else. I put it in common.py mainly because https://github.com/ansible/ansible/pull/61201/files#diff-f4cdbf8615132076d5987c273fb12308R856 instantiates it. I guess I could move that method over into the docker_login module, if that would be preferred.

Reply:
Yep, that would be preferred. `module_utils` should only contain code used by multiple modules (or very like to be used with multiple modules).